src/utilities/maybeExpand.ts

```diff
@@ -3,6 +3,8 @@ import { FileCollectionItem } from '../FileCollectionItem';
 import { fileCollectionItemsUngzip } from '../fileCollectionItemsUngzip';
 import { fileCollectionItemsUnzip } from '../fileCollectionItemsUnzip';
 
+import { FilterOptions } from './maybeFilter';
+
```

Comment:
I don't think that this method needs to filter the data. Is it not enough to filter at the end when everything is expanded ?

Reply:
And is it a problem if we remove always the '.' files ?